lms/envs/aws.py

```diff
@@ -333,6 +333,7 @@
 COMMENTS_SERVICE_KEY = ENV_TOKENS.get("COMMENTS_SERVICE_KEY", '')
 CERT_QUEUE = ENV_TOKENS.get("CERT_QUEUE", 'test-pull')
 ZENDESK_URL = ENV_TOKENS.get('ZENDESK_URL', ZENDESK_URL)
+ZENDESK_CUSTOM_FIELDS = ENV_TOKENS.get('ZENDESK_CUSTOM_FIELDS', ZENDESK_CUSTOM_FIELDS)
```

Comment:
My understanding of this is that we attempt to load environment-specific config from ENV_TOKENS and if it's not found, fall back to the default config specified in `lms/envs/common.py`. Is this correct?